services/server/src/server/services/storageServices/AbstractDatabaseService.ts

```diff
@@ -51,6 +53,9 @@ export default abstract class AbstractDatabaseService {
     this.postgresDatabase = options.postgres?.database;
     this.postgresUser = options.postgres?.user;
     this.postgresPassword = options.postgres?.password;
+    if (options.schema) {
+      this.schema = options.schema;
+    }
```

Comment:
```suggestion
    } else {
      logger.warn(`Database schema name not set, using default "public"`, { identifier: this.IDENTIFIER, schema: options.schema })
    }
```

Reply:
Abstract properties cannot be accessed in the constructor, I'll log the schema somewhere else